### DISCUSSION TITLE
Quasar Meta problem

### DISCUSSION THREAD

**Discussion by boo01** [quasarframework/quasar#9786] [General - Components / Directives / etc] [answered]
Hello,
I have problem with metas. I crated Metas with quasar plugin, it shows everything when I open inspect element but facebook cant read it. 
please help.
I import Meta like this:
import { openURL, AddressbarColor, Meta, colors, Notify, date, Loading, QSpinnerFacebook } from "quasar";
Here is my code : 

  meta () {
    return {
     // sets document title
    title: this.infos.name,
    // optional; sets final title as "Index Page - My Website", useful for multiple level meta
    titleTemplate: title => ${title},

    // meta tags
    meta: {
      description: {
        name: "description",
        content: this.infos.meta_desc
      },
      keywords: {
        name: "keywords",
        content: this.infos.meta_key
      },
      equiv: {
        "http-equiv": "Content-Type",
        content: "text/html; charset=UTF-8"
      },
      // note: for Open Graph type metadata you will need to use SSR, to ensure page is rendered by the server
      ogTitle: {
        property: "og:title",
        // optional; similar to titleTemplate, but allows templating with other meta properties
        content: this.infos.name,
      },
      ogImage: {
        property: "og:image",
        // optional; similar to titleTemplate, but allows templating with other meta properties
        content: this.uplUrl + this.infos.img,
      
      },
      ogDescription: {
        property: "og:description",
        content: this.infos.meta_desc
      },
    }
    }
  },

**Comment by hawkeye64** [comment, marked as answer]
In order to get true meta, you have to be running an SSR server, where, when requested by a bot, it generates the necessary meta using JavaScript and then sends the page already done. If you are not using SSR, then you have an SPA. In which case, a site like Facebook retrieves your page but doesn't run the JavaScript necessary for you to have your meta in place.
